### PR TITLE
ffmpeg: remove unused texi2html dependency

### PR DESCRIPTION
**Pull request by ddelange**
Ref https://github.com/Homebrew/homebrew-core/issues/60409

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by jonchang** [comment]
* license should be `GPL-2.0-or-later`
* you can use `brew audit --fix ffmpeg` to automatically fix many issues (including method ordering). but in this case I think we can avoid changing the revision altogether since it's only a build-time dependency
* texinfo is actually shipped with macOS so let's try removing it altogether and see if it still builds

**Comment by ddelange** [comment]
@jonchang Tests are passing here :) However, on my other PR the tests are failing for ubuntu (https://github.com/ddelange/homebrew-ffmpeg/pull/1). Somehow, perl tests are failing, subsequently failing the `brew test-bot` command. Any idea why perl would not build on ubuntu? And how would you find out which (recursive) dependency needs `perl`?

File: Formula/ffmpeg.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@ class Ffmpeg < Formula
   homepage "https://ffmpeg.org/"
   url "https://ffmpeg.org/releases/ffmpeg-4.3.1.tar.xz"
   sha256 "ad009240d46e307b4e03a213a0f49c11b650e445b1f8be0dda2a9212b34d2ffb"
-  # None of these parts are used by default, you have to explicitly pass `--enable-gpl`
-  # to configure to activate them. In this case, FFmpeg's license changes to GPL v2+.
-  license "GPL-2.0"
+  # FFmpeg's license changes to GPL v2+, as --enable-gpl is always passed.
```
Comment (ddelange) [inline Formula/ffmpeg.rb:6]: shall i also link to `https://ffmpeg.org/legal.html` in this comment?

Comment (ddelange) [reply Formula/ffmpeg.rb:6]: could even add something like `to build FFmpeg without this flag (resulting LGPLv2.1+ license), use homebrew-ffmpeg/ffmpeg/ffmpeg`

Comment (SMillerDev) [reply Formula/ffmpeg.rb:6]: Please just change the license identifier.

Comment (ddelange) [reply Formula/ffmpeg.rb]: @SMillerDev sure, reverted 🤙

**Comment by SMillerDev** [comment]
If this builds please squash your commits into one with the PR title as the commit title.

**Review by jonchang — APPROVED** [review]
Thanks for contributing to Homebrew, @ddelange!

**Pull request closed without merging**